tools/clang/test/CodeGenSPIRV/intrinsics.modf.swizzle.hlsl

```diff
@@ -0,0 +1,16 @@
+// Run: %dxc -T vs_6_0 -E main
+
+// CHECK: [[glsl:%\d+]] = OpExtInstImport "GLSL.std.450"
+
+void main() {
+// CHECK:        [[value_v2f:%\d+]] = OpLoad %v2float %v2f
+// CHECK-NEXT:        [[modf:%\d+]] = OpExtInst %ModfStructType [[glsl]] ModfStruct [[value_v2f]]
+// CHECK-NEXT:     [[modf_ip:%\d+]] = OpCompositeExtract %v2float [[modf]] 1
+// CHECK-NEXT: [[modf_ip_int:%\d+]] = OpConvertFToS %v2int [[modf_ip]]
+// CHECK-NEXT:     [[swizzle:%\d+]] = OpLoad %v3int %v3i
+// CHECK-NEXT:      [[vector:%\d+]] = OpVectorShuffle %v3int [[swizzle]] [[modf_ip_int]] 3 4 2
```

Comment:
use {{%\d+}} instead of  [[modf_ip_int]]

Reply:
done